tests/org.eclipse.n4js.ide.tests/.gitignore

```diff
@@ -1 +1,2 @@
 test-catalog.json
+test-data/
```

Comment:
I know that this wasn't changed in this PR, but:

The name "test-data" for the temporary folder where the projects etc are created is a bit unfortunate, because often the folder where source test files are located are called "testdata" or the like. I guess you did not want to use a temporary folder at the usual OS-specific location to make it easier to debug it.

What about "test-workspace"? Or do you have a better idea?

Reply:
You're right I did not use the OS specific location to make debugging easier.

Besides, yes, I rename it.